docs/source/tutorials_source/package/tutorial_timm_backbone.py

```diff
@@ -0,0 +1,91 @@
+"""
+.. _lightly-timm-backbone-tutorial-8:
+
+Tutorial 8: Using timm models as backbones
+===========================================
+
+You can use LightlySSL to pre-train any model using self-supervised learning since they
+share a similar workflow. All methods have a base model (the backbone), which can be any
+fundamental model such as ResNet or MobileNet.
+
+In this tutorial, we will learn how to use a model architecture from the timm library
+as a backbone in a self-supervised learning workflow.
+"""
+
+# %%
+# Imports
+# -------
+#
+# Import the Python frameworks we need for this tutorial.
+# Make sure you have the necessary packages installed.
+#
+# .. code-block:: console
+#
+#   pip install lightly"[timm]"
+
+
+import timm
+import torch
+import torch.nn as nn
+
+# %%
+# timm comes packaged with >700 pre-trained models designed to be flexible and easy to use.
+# These pre-trained models can be loaded using the
+# `create_model() <https://huggingface.co/docs/timm/v1.0.8/en/reference/models#timm.create_model>`_
+# function. For example, we can use the following snippet to create an efficient model.
+
+model = timm.create_model(model_name="efficientnet_b0")
+
+# %%
+# To use a model from timm as a backbone for a self supervised learning method
+# we need to remove the last linear layer. We can use the children helper method
+# to break down a model into its constituent layers and then convert into a list
+# to filter out the necessary layers
+
+backbone = nn.Sequential(
+    *list(model.children())[:-1],
+)
```

Comment:
I think we should use `model.forward_features` when pretraining timm models. The `nn.Sequential` trick is quite hacky and it makes it hard to save and load the pretrained model again.